### PR TITLE
Add url support in env set command

### PR DESCRIPTION
**Pull request by ManishPrajapati-GoFynd**

### CODE REVIEW

File: src/lib/Env.ts [hunk 1]
```diff
@@ -106,6 +117,26 @@ export default class Env {
...
+          const response = await axios.get(url)
+          
+          if(response?.data?.ok){
+            Env.setEnv(options.url);
+            Logger.info(`Env set to: ${chalk.bold(options.url)}`);
+          } else {
+            throw new Error("Provided url is not sandbox URL.")
+          }
+        }catch(err){
+          console.log(err);
```
Comment (vivek8690) [inline src/lib/Env.ts]: Don't use console log @manishprajapati52

File: src/lib/Env.ts [hunk 2]
```diff
@@ -106,6 +117,26 @@ export default class Env {
...
+          throw new Error("Provided url is not sandbox URL.")
```
Comment (vivek8690) [inline src/lib/Env.ts]: use `CommandError` instead of  `Error` @manishprajapati52

File: src/lib/Env.ts [hunk 3]
```diff
@@ -106,6 +117,26 @@ export default class Env {
...
+
+        try{
+          const url = urljoin("https://",options.url, '/service/application/content/_healthz')
+          const response = await axios.get(url)
+          
+          if(response?.data?.ok){
+            Env.setEnv(options.url);
+            Logger.info(`Env set to: ${chalk.bold(options.url)}`);
+          } else {
+            throw new Error("Provided url is not sandbox URL.")
```
Comment (vivek8690) [inline src/lib/Env.ts]: Use CommandError instead of Error @manishprajapati52

**Review by vivek8690 — CHANGES_REQUESTED** [review]
Add commented changes @manishprajapati52

File: src/lib/Env.ts (hunk 2 above)
Comment (ManishPrajapati-GoFynd) [reply src/lib/Env.ts]: We have catch outside that, there we are catching this error and Passing into CommandError.

File: src/lib/Env.ts (hunk 3 above)
Comment (ManishPrajapati-GoFynd) [reply src/lib/Env.ts]: Didn't get this...
Don't use CommandError?

**Review by vivek8690 — APPROVED** [review]

**Pull request merged**